cli/ts/commands/genProofs.ts

```diff
@@ -248,14 +248,14 @@ export const genProofs = async ({
       signer.provider!,
       await maciContract.getAddress(),
       coordinatorKeypair,
-      pollId,
+      Number(pollId),
       fromBlock,
       blocksPerBatch,
       endBlock,
     );
   }
 
-  const poll = maciState!.polls[pollId];
+  const poll = maciState!.polls[Number(pollId)];
```

Comment:
Is it possible to get rid of such casts? Maybe it makes sense to update `polls` property so it will be `Record<bigint, Poll>`?

Reply:
love this, will try

Reply:
seems like you cannot use bigint as a key type for record

Reply:
I'll use a Map if that sounds good with you too?

Reply:
Yes, Map is a good option too.